Source/WebCore/platform/ScrollableArea.cpp

```diff
@@ -172,6 +172,15 @@ void ScrollableArea::scrollToPositionWithoutAnimation(const FloatPoint& position
     scrollAnimator().scrollToPositionWithoutAnimation(position, clamping);
 }
 
+void ScrollableArea::scrollToPositionWithoutAnimation(ScrollbarOrientation orientation, float offset)
```

Comment:
Since this function isn't about scrolling to an offset, consider changing the argument name:
```suggestion
void ScrollableArea::scrollToPositionWithoutAnimation(ScrollbarOrientation orientation, float position)
```